3.0.0RC1/airmet.xsd

```diff
@@ -106,24 +106,10 @@ YUDD AIRMET A3 VALID ...</documentation>
 							<documentation>The reported phenomenon, such as thunderstorm, tropical cyclone, icing, mountain wave, etc.</documentation>
 						</annotation>
 					</element>
-					<element name="analysis" minOccurs="0" maxOccurs="1">
+					<element name="analysis" type="iwxxm:AIRMETEvolvingConditionCollectionType" minOccurs="0" maxOccurs="1">
```

Comment:
It is interesting to know that nobody found this bug since the publishing of IWXXM 1.x.  :)
Anyway I have just committed the revised schemas for further checking.